<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Adminedit_apt</class>
 <widget class="QDialog" name="Adminedit_apt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="choice_box">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>541</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Drop your Choice</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Choice 4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="choice_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Choice 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="choice_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="choice_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="choice_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Choice 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Choice 1</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="qn_box">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>541</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Drop your Question</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Write Queston</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qn_editor">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Qn.no :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qn_number">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="submitbtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>560</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>580</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>DB Status :</string>
   </property>
  </widget>
  <widget class="QLabel" name="qn_db_status">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>580</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>560</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
